### PR TITLE
Update cookie policy 2

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Update cookie-policy version to 3.0.0
- I couldn't push to Beth's branch, so this superceeds it

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- In browser settings remove cookies, refresh page and check cookie policy needs to be explicitly accepted

## Issue / Card

Fixes [#64](https://github.com/canonical-web-and-design/cookie-policy/issues/64)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +33,6 @@
     "*.scss": "dotrun exec yarn run lint-scss",
     "*.py": "dotrun exec yarn run lint-python",
     "*.js": [
-      "dotrun exec yarn run lint-js",
```
Comment (anthonydillon) [inline package.json:35]: Why remove the lint here?

**Review by anthonydillon — COMMENTED** [review]
Code looks good with a comment.

File: package.json [hunk 2]
```diff
@@ -32,7 +33,6 @@
     "*.scss": "dotrun exec yarn run lint-scss",
     "*.py": "dotrun exec yarn run lint-python",
     "*.js": [
```
Comment (pmahnke) [inline package.json:35]:
```suggestion
    "*.js": [
      "dotrun exec yarn run lint-js",
```

File: package.json (hunk 1 above)
Comment (pmahnke) [reply package.json]: fixed, pretty sure I didn't do that.

**Review by anthonydillon — APPROVED** [review]
QA worked for me

**Pull request merged**